Objects/abstract.c

```diff
@@ -827,7 +827,7 @@ binary_op(PyObject *v, PyObject *w, const int op_slot, const char *op_name)
             PyErr_Format(PyExc_TypeError,
                 "unsupported operand type(s) for %.100s: "
                 "'%.100s' and '%.100s'. Did you mean \"print(<message>, "
-                "file=<output_stream>)\"",
+                "file=<output_stream>)?\"",
```

Comment:
The question mark should be outside the quote marks, since it isn't part of the suggested Python code.